install

```diff
@@ -125,8 +115,7 @@ do_install()
   rosinstall_generator ${ROS_CONFIGURATION} ${ROS_EXTRA_PACKAGES} --rosdistro ${ROS_DISTRO} --deps --tar > ${WS}.rosinstall
   wstool init -j8 src ${WS}.rosinstall
 
-  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+. Although formulae and patches
-  # exist to build Ogre 1.9, rviz, and Gazebo together, I have found this approach to be the most solid and consistent.
+  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+.
   if [ -d src/rviz ]; then
     pushd src/rviz/ogre_media/models
     curl https://raw.githubusercontent.com/ros-visualization/rviz/hydro-devel/ogre_media/models/rviz_cone.mesh > rviz_cone.mesh
```

Comment:
The Ogre 1.9 formula doesn't look too bad— it's really just two brief patches: https://github.com/osrf/homebrew-simulation/pull/24/files#diff-0bfad96e7da860a50fe01709e5f95ea0

The big questions from my side would then be:
- Would you be willing for `osrf/simulation/ogre` to become Ogre 1.9, or would we want it on a versioned dep?
- What would be story for bottled Gazebo?

The ideal from my side here would obviously be to have `ros-install-osx` depend on an Ogre and Gazebo version that are both bottled, and have rviz build from source against the same Ogre version. I can do rosdep overrides and so-on handle the rviz side of it, but I'd like to take advantage of whatever you're able to bottle.